README.md

```diff
@@ -59,11 +59,11 @@ To work on the document on your machine, you need the following:
 To build the document, either run this in the command line:
 
 ```bash
-latexmk -pdf -silent -shell-escape "main"
+latexmk -pdf -silent -shell-escape "main_en"
```

Comment:
would it be worth adding a comment that the en part might be substituted for non-techy people?

Reply:
Yes, definitely. I'm thinking of revamping the readme totally, but in a separate PR.